test/decl/typealias/generic.swift

```diff
@@ -1,9 +1,31 @@
-// RUN: %target-parse-verify-swift
+// RUN: %target-parse-verify-swift -enable-experimental-nested-generic-types
 
-struct MyType<TyA, TyB> {
+struct MyType<TyA, TyB> { // expected-note {{declared here}}
   var a : TyA, b : TyB
 }
 
+//
+// Type aliases that reference unbound generic types -- not really generic,
+// but they behave as such, in the sense that you can apply generic
+// arguments to them.
+//
+
+// FIXME: This should work?
+typealias OurType = MyType  // expected-error {{reference to generic type 'MyType' requires arguments in <...>}}
```

Comment:
I think it is right to reject this, we want a declaration to be obvious that it is generic, and state its type parameters.